src/parser/Lexer.cpp

```diff
@@ -836,6 +838,11 @@ Scanner::ScanIDResult Scanner::getComplexIdentifier()
     }
 
     String* str = new UTF16String(id.data(), id.length());
+
+    if (UNLIKELY(this->parserContext->await && id == u"await")) {
+        this->throwUnexpectedToken(Messages::KeywordMustNotContainEscapedCharacters);
```

Comment:
It is not different, but we should check this case when creating identifier every time
so it can ruins performance but this escaped case is very rare & getComplexIdentifier also rare

Reply:
Ok I see.